tests/testthat/test-variable-order.R

```diff
@@ -712,134 +667,35 @@ with_mock_crunch({
-                "        starttime",
-                "    starttime",
-                "Gender",
-                "Text variable ftw",
-                "Birth Year",
-                sep = "\n"
-            ),
-            fixed = TRUE
-        )
-    })
-    test_that("dedupeOrder removes duplicate entries", {
-        expect_prints(dedupeOrder(ord),
-            paste("[+] Alpha",
-                "    [+] Bravo",
-                "        [+] Charlie",
-                "            Gender",
-                "            Birth Year",
-                "        mymrset",
-                "[+] Delta",
-                "    [+] Echo",
-                "        [+] Foxtrot",
-                "            Cat Array",
-                "        starttime",
-                "Text variable ftw",
-                sep = "\n"
-            ),
-            fixed = TRUE
-        )
-    })
-    test_that("dedupeOrder doesn't mutate an order that's already deduped", {
-        expect_prints(dedupeOrder(dedupeOrder(ord)),
-            paste("[+] Alpha",
-                "    [+] Bravo",
-                "        [+] Charlie",
-                "            Gender",
-                "            Birth Year",
-                "        mymrset",
-                "[+] Delta",
-                "    [+] Echo",
-                "        [+] Foxtrot",
-                "            Cat Array",
-                "        starttime",
-                "Text variable ftw",
-                sep = "\n"
-            ),
-            fixed = TRUE
-        )
-    })
-    test_that("Setting duplicates <- FALSE triggers dedupeOrder", {
-        duplicates(ord) <- FALSE
-        expect_prints(ord,
-            paste("[+] Alpha",
-                "    [+] Bravo",
-                "        [+] Charlie",
-                "            Gender",
-                "            Birth Year",
-                "        mymrset",
-                "[+] Delta",
-                "    [+] Echo",
-                "        [+] Foxtrot",
-                "            Cat Array",
-                "        starttime",
-                "Text variable ftw",
-                sep = "\n"
-            ),
-            fixed = TRUE
+    test_that("dedupeOrder is deprecated", {
+        expect_deprecated(
+            expect_prints(dedupeOrder(ord),
+                paste("[+] Demos",
+                    "    [+] Others",
+                    "        Text variable ftw",
+                    "        starttime",
+                    "    [+] Arrays",
+                    "        Cat Array",
+                    "        [+] MR",
+                    "            mymrset",
+                    "        Birth Year",
+                    "        Gender",
+                    sep = "\n"
+                ),
+                fixed = TRUE
+            )
         )
     })
+
     test_that("intersect_entities", {
         expect_prints(intersect_entities(ord, ds[c("birthyr", "starttime")]),
```

Comment:
This is an example of where the vdiffr package might be helpful because it makes it easier to verify that the current printing expectations are correct.